### ISSUE TITLE
generate-example script for terra-icon failing

### ISSUE THREAD

**Issue by eawww** [cerner/terra-core#3571]
# Bug Report

## Description

`generate-example` script for `terra-icon` fails with 
```
> node scripts/lib/generate-example/index.js

events.js:291
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '../../packages/terra-icon/src/terra-dev-site/doc/example/IconThemeable.jsx'
Emitted 'error' event on WriteStream instance at:
    at internal/fs/streams.js:361:12
    at FSReqCallback.oncomplete (fs.js:156:23) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '../../packages/terra-icon/src/terra-dev-site/doc/example/IconThemeable.jsx'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! terra-icon@3.46.1 generate-example: `node scripts/lib/generate-example/index.js`
npm ERR! Exit status 1
```
Generated files were migrated to [packages/terra-core-docs/src/terra-dev-site/doc/icon/example/](https://github.com/cerner/terra-core/tree/main/packages/terra-core-docs/src/terra-dev-site/doc/icon/example), but the path in the scripts is still the old one:
[generateStaticIcons.js](https://github.com/cerner/terra-core/blob/dd13bf9bee4dd05faa62cebbe0929734edac2086/packages/terra-icon/scripts/src/generate-example/generateStaticIcons.js#L4)
[generateThemeableIcons.js](https://github.com/cerner/terra-core/blob/main/packages/terra-icon/scripts/src/generate-example/generateThemeableIcons.js)

## Steps to Reproduce

1. Run `npm run migrate-cerner-one-icons` in `terra-icon`

## Expected Behavior

Generated scripts are written to their home in `terra-core-docs`

## Possible Solution

Update path in lines linked above to `packages/terra-core-docs/src/terra-dev-site/doc/icon/example/...`

## Environment

* Component Name and Version: terra-icon v3.46.1
* Browser Name and Version: n/a
* Node/npm Version: node v12.22.9/npm v6.14.15
* Webpack Version: 
* Operating System and version (desktop or mobile): MacOS 12.1 Monterey

**Comment by eawww** [comment]
Note: this overlaps with work we'll be doing on the icon docs page in the next few weeks. It can be resolved alongside that if nobody thinks it's more urgent.
